packages/@okta/vuepress-site/docs/guides/implement-grant-type/main/interactioncode/use-flow.md

```diff
@@ -1,26 +1,26 @@
-The Interaction Code flow is intended for developers who want to control the remediation user experience without redirecting the authentication experience to Okta. The Interaction Code flow is used in the [embedded authentication deployment model](/docs/concepts/redirect-vs-embedded/#embedded-authentication) where you can manage the user interaction by leveraging either an Identity Engine SDK or the Sign-In Widget with an Identity Engine SDK.
+The Interaction Code flow is intended for developers who want to embed the authentication process in their apps rather than redirect it to Okta. Implement it in the [embedded authentication deployment model](/docs/concepts/redirect-vs-embedded/#embedded-authentication) using an Identity Engine SDK (IDX SDK) either alone or with the Sign-In Widget.
 
 ### Use an SDK
 
-Okta recommends using an Identity Engine SDK with the Sign-In Widget to implement your embedded authentication Interaction Code flow. See [Languages and SDKs](/code/) for information on Identity Engine SDKs that you can download to start using with your app.
+Okta recommends using an IDX SDK with the Sign-In Widget to implement your embedded authentication Interaction Code flow. See [Languages and SDKs](/code/) for information on IDX SDKs that you can download to start using with your app.
 
-> **Note:** Even if you are not using the Sign-In Widget in your final embedded app, the Sign-In Widget can be used to validate and test your defined policies before investing a lot of time on coding the authentication flow.
+> **Note:** Consider using the Sign-In Widget to test your authentication policies even if you don’t plan to use it in production. It’s quicker to embed it in a test app than code the whole flow from scratch.
 
-For instructions on how to install and use Okta Identity Engine SDKs, refer to [Download and set up the SDK, Sign-In Widget, and sample app](/docs/guides/oie-embedded-common-download-setup-app/). You can download Okta sample apps to see how the SDKs are used in your app's framework. See [Examples](#examples) for a list of sample apps.
+For instructions on how to install and use IDX SDKs, refer to [Download and set up the SDK, Sign-In Widget, and sample app](/docs/guides/oie-embedded-common-download-setup-app/). Each IDX SDK contains its own example applications with embedded authentication. See [Examples](#examples).
```

Comment:
```suggestion
For instructions on how to install and use Identity Engine SDKs, see [Download and set up the SDK, Sign-In Widget, and sample app](/docs/guides/oie-embedded-common-download-setup-app/). Each Identity Engine SDK contains its own example applications with embedded authentication. See [Examples](#examples).
```

Reply:
Gosh - how did that "refer to" get left in?